django/core/management/commands/dumpdata.py

```diff
@@ -6,6 +8,38 @@
 from django.core.management.utils import parse_apps_and_model_labels
 from django.db import DEFAULT_DB_ALIAS, router
 
+try:
+    import bz2
+    HAS_BZ2 = True
```

Comment:
Should we mirror how this is implemented in `loaddata` and use lowercase here?

```suggestion
    has_bz2 = True
```

Same for `HAS_LZMA` → `has_lzma`.

Reply:
Fixed